OpenShift/helm/nexus-iq/templates/deployment.yaml

```diff
@@ -23,9 +23,9 @@ spec:
         image: {{ .Values.iq.imageName }}:{{ .Values.iq.imageTag }}
         ports:
         - containerPort: {{ .Values.iq.applicationPort }}
-        {{- if .Values.deployment.terminationGracePeriodSeconds }}
-        terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
-        {{- end }}
+        # {{- if .Values.deployment.terminationGracePeriodSeconds }}
+        # terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
+        # {{- end }}
```

Comment:
interesting. i had also found this setting to be problematic in the operator code, and had to remove reference to it in the values.

Reply:
if this isn't useful, then please delete it. Commented out code creates a lot of clutter

Reply:
@jflinchbaugh why was this a problem? I believe I added it in because of some specific concerns that the shutdown would not be done in time (which was something I experienced personally). It is optional so I don't see the value in removing this support.

Reply:
It is useful on OCP, unsupported on k8s